File: wizards/com/sun/star/wizards/letter/CGLetterWizard.java
package com.sun.star.wizards.letter;

import com.sun.star.wizards.common.ConfigGroup;

public class CGLetterWizard extends ConfigGroup {

    public int cp_LetterType;
    public CGLetter cp_BusinessLetter = new CGLetter();
    public CGLetter cp_PrivateOfficialLetter = new CGLetter();
    public CGLetter cp_PrivateLetter = new CGLetter();

}

File: wizards/com/sun/star/wizards/letter/CGLetterWizard.py
from common.ConfigGroup import *
from CGLetter import CGLetter

class CGLetterWizard (ConfigGroup):

    def __init__(self):
        self.cp_LetterType = int()
        self.cp_BusinessLetter = CGLetter()
        self.cp_PrivateOfficialLetter = CGLetter()
        self.cp_PrivateLetter = CGLetter()
